tests/route_check_test_data.py

```diff
@@ -881,5 +892,40 @@
         },
         RET: -1,
     },
-
+    "21": {
+        DESCR: "basic route check, good one, bgp disabled",
+        ARGS: "route_check -m INFO -i 1000",
+        PRE: {
+            APPL_DB: {
```

Comment:
You need to pass namespace here? Also need to pass    MULTI_ASIC flag and namespace list..
have a look at other testcases..